webrecorder/browsertrix
# Add exclusion to running crawl

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/319

### Manual testing
Pre-req: Run `yarn` to get new dependencies
1. Run app with `yarn start` and navigate to crawls
2. Start a crawl
3. Go to crawl detail -> Exclusions tab
4. Type in an exclusion. Verify "Pending Exclusions" and "Crawl Queue" lists update
5. Submit with "Add Exclusion". Verify exclusion table and lists update

### Screenshots
<img width="814" alt="Screen Shot 2022-11-01 at 3 40 41 PM" src="https://user-images.githubusercontent.com/4672952/199356210-3eba1bcc-09dc-4349-b208-460a6e0c5549.png">

<img width="637" alt="Screen Shot 2022-11-08 at 11 09 26 AM" src="https://user-images.githubusercontent.com/4672952/200630620-f8b73351-b296-4783-be07-2a2ebd5fc2d2.png">
